app/controllers/api/fixed/active/sessions_controller.rb

```diff
@@ -17,7 +17,13 @@ def index
 
         def index2
           GoogleAnalyticsWorker::RegisterEvent.async_call('Fixed active sessions#index2')
-          result = Api::ToActiveSessionsJson.new(form: form).call
+          # splitting the logic for governemnt data, to improve performance
+          sensor_name = form.to_h.to_h[:sensor_name]
+          if sensor_name == 'government-pm2.5' || sensor_name == 'government-no2' || sensor_name == 'government-o3'
+            result = ::FixedSessions::IndexInteractor.new(form: form).call
+          else
+            result = Api::ToActiveSessionsJson.new(form: form).call
+          end
```

Comment:
I guess it's not very important, but I wonder if it should be split here or in the interactor.
Right now we have some duplication, in both services we check if the form is valid:
`return Failure.new(form.errors) if form.invalid?`